README.md

```
@@ -1 +1,176 @@
+
+**Gameplay:**
+1. Select one of your pieces to see valid moves
+2. Choose a move using one of your cards
+3. After moving, exchange the used card with the side card
+4. The side card goes to your opponent on their next turn
+
+For a detailed tutorial, visit the "How to Play" page in the app.
+
+## API Endpoints
+
+### Authentication
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)